playground/test-utils.ts

```diff
@@ -325,3 +325,15 @@ export async function killProcess(
     serverProcess.kill('SIGTERM', { forceKillAfterTimeout: 2000 })
   }
 }
+
+export const toContain = (actuals: any[], matcher: string) => {
+  expect(actuals).toEqual(
+    expect.arrayContaining([expect.stringMatching(matcher)]),
+  )
+}
+
+export const notContain = (actuals: any[], matcher: string) => {
+  expect(actuals).not.toEqual(
+    expect.arrayContaining([expect.stringMatching(matcher)]),
+  )
+}
```

Comment:
I think we better inline these instead of creating the abstractions. Even if they are more verbose, it is easier when parsing the test spect to know where the `expect`s and what they are testing.